packages/kbn-apm-synthtrace/src/cli/scenario.ts

```diff
@@ -16,7 +16,15 @@ type Generate<TFields> = (options: {
   range: Timerange;
 }) => SynthtraceGenerator<TFields> | Array<SynthtraceGenerator<TFields>> | Readable;
 
+export interface ScenarioOptions {
+  // Equivalent of APM Server Setting - Transactions.MaxTransactionGroups
+  max_transactions?: number;
+  // Equivalent of APM Server Setting - Transactions.MaxServices
+  max_services?: number;
```

Comment:
can you make the naming more specific here, ie that it is clear that it represents overflow settings?